pkgs/development/python-modules/trafilatura/default.nix

```
@@ -55,14 +62,16 @@ buildPythonPackage rec {
     substituteInPlace tests/cli_tests.py --replace "trafilatura_bin = 'trafilatura'" "trafilatura_bin = '$out/bin/trafilatura'"
   '';
 
-  pythonImportsCheck = [ "trafilatura" ];
+  pythonImportsCheck = [
+    "trafilatura"
+  ];
 
   meta = with lib; {
     description = "Python package and command-line tool designed to gather text on the Web";
-    mainProgram = "trafilatura";
     homepage = "https://trafilatura.readthedocs.io";
     changelog = "https://github.com/adbar/trafilatura/blob/v${version}/HISTORY.md";
     license = licenses.gpl3Plus;
```

License was changed to Apache 2.0 in version 1.8.0.

```suggestion
    license = licenses.asl20;
```